src/app/components/options-sidebar/time-clock/time-clock.component.ts

```diff
@@ -43,73 +43,73 @@ export class TimeClockComponent  implements OnInit {
     this.hour = 0;
     this.minute = 0;
     this.seconds = 0;
-   }
-
-   employeClockIn(): boolean {
-     this.isClockInEnable = true;
-     this.isClockIn = !this.isClockIn;
-     this.isHidenForm = false;
-     this.startTimer();
-     this.setArrivalAndDepartureTimes();
-     return this.isClockIn;
-   }
-
-   employeClockOut() {
-     if ( this.isEnterTechnology === false ) {
-       this.isClockIn = false;
-       this.showAlertEnterTecnology = true;
-     } else {
-       this.setDefaultValuesToFields();
-       this.pauseTimer();
-       this.setArrivalAndDepartureTimes();
-     }
-   }
-
-   enterTechnology(data: string) {
-     if ( data.length > 0 ) {
+  }
+
+  employeClockIn(): boolean {
+    this.isClockInEnable = true;
+    this.isClockIn = !this.isClockIn;
+    this.isHidenForm = false;
+    this.startTimer();
+    this.setArrivalAndDepartureTimes();
+    return this.isClockIn;
+  }
+
+  employeClockOut() {
+    if (this.isEnterTechnology === false) {
```

Comment:
Since users can enter more than one technologies, you should rename this var to:
this.areTechnologiesEntered